### PR TITLE
improve debugger

### PR DESCRIPTION
**Pull request by bognari**
This will hopefully help to make the debug font more readable on height resolutions with small displays.

partial fix / workaround for https://github.com/korlibs/korge/issues/556

### CODE REVIEW

File: korge/src/commonMain/kotlin/com/soywiz/korge/view/Views.kt
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/view/Views.kt:214]: Instead of passing it as a primitive as an extra argument, can we put this `extraFontScale` inside the `RenderContext`? Either as a property or as an extrinsic Extra.Property? Maybe with a name like `debugExtraFontScale` or something?

Additionally, are you doing this because we are not properly detecting the device pixel ratio on windows? Could we implement that instead?

**Comment by bognari** [comment]
Hey @soywiz I did you requestet change and insertet the debugExtraFontScale  into the RendererContext.

Maybe you are right and it would be a better idea to implement the device pixel ratio correctly but maybe for someone the font is still to small so it is possible to make the font bigger (or smaller).

**Comment by bognari** [comment]
ok now we use the scaling settings too. 
https://stackoverflow.com/questions/32586883/windows-scaling

**Comment by soywiz** [comment]
Awesome! Thanks :) let's wait for CI to finish

File: korgw/src/jvmMain/kotlin/com/soywiz/korgw/awt/AwtAg.kt
Comment (soywiz) [inline korgw/src/jvmMain/kotlin/com/soywiz/korgw/awt/AwtAg.kt:40]: Tested on mac. On mac, it should be something like `scale / pixelsPerLogicalInchRatio`.

But I believe we should separate those values:

```
override val devicePixelRatio: Double get() = ... scaleX
override val pixelsPerLogicalInchRatio: Double get() = ... screenResolution / defaultPixelsPerLogicalInch
// Use this in the debug handler, while allowing people to access raw devicePixelRatio without the noise of window scaling
override val computedPixelRatio: Double get() = devicePixelRatio / pixelsPerLogicalInchRatio
```

Otherwise `devicePixelRatio` stops being true. Is it 1 on your hdpi screen? On mac devices with retina displays it is 2, without it, it is 1.

Comment (bognari) [reply korgw/src/jvmMain/kotlin/com/soywiz/korgw/awt/AwtAg.kt:40]: But yes we can separate the values.

Comment (bognari) [reply korgw/src/jvmMain/kotlin/com/soywiz/korgw/awt/AwtAg.kt]: i'm still not sure about the calculation of "computedPixelRatio" becasue in my opinion it must be 

`devicePixelRatio * pixelsPerLogicalInchRatio` 
or
`pixelsPerLogicalInchRatio / devicePixelRatio `

**Comment by bognari** [comment]
@soywiz any feedback about my pr?

**Comment by soywiz** [comment]
Yes, sorry for the delay. Had a pretty long couple of weeks, and forgot to check this. Im not at home right now, but Ill check it asap

**Comment by bognari** [comment]
sorry. all is good. take your time. this was just a reminder

File: korge/src/commonMain/kotlin/com/soywiz/korge/render/RenderContext.kt
Comment (soywiz) [inline korge/src/commonMain/kotlin/com/soywiz/korge/render/RenderContext.kt:50]: Downloaded the PR, and this line seems to fail because RGBA is not found.
You should add a `import com.soywiz.korim.color.*`

**Review by soywiz — APPROVED** [review]
Sorry for the delay.

I have tried it and seems to work properly, though you should add a `import com.soywiz.korim.color.*` for it to compile.

Once you add that import, let's merge this ^^

**Comment by soywiz** [comment]
Thanks!

**Pull request merged**
